alerta/app/database.py

```diff
@@ -1108,6 +1108,40 @@ def is_user_valid(self, id=None, name=None, login=None):
         if login:
             return bool(self._db.users.find_one({"login": login}))
 
+
+    def update_user(self, login, fields=None):
```

Comment:
This method needs to update the user by `id` not `login`. I suggest the method signature should look like this...

```
def update_user(self, id, name, login, password=None, provider="", text="", email_verified=False):
```